packages/@lwc/engine-server/src/__tests__/fixtures/render-correct-closing-tags/expected.html

```diff
@@ -0,0 +1,24 @@
+<x-elements>
+  <template shadowrootmode="open">
+    <a href="https://www.salesforce.com/">
+    </a>
+    <label for="textInput">
+      Some input
+    </label>
+    <input id="textInput" type="text">
+    <svg>
+      <pattern>
+        <image xlink:href="https://www.salesforce.com/"/>
+      </pattern>
+      <image xlink:title="title"/>
+    </svg>
+    <svg>
+      <pattern>
+        <image xlink:href="https://www.salesforce.com/">
+        </image>
+      </pattern>
+      <image xlink:title="title">
+      </image>
+    </svg>
```

Comment:
Rendered from template and rendered from `lwc:inner-html` have different output. I assume that's what engine-server does, so it's fine, but is that something we'd want to fix?